.github/workflows/manage-example-branches.yml

```
@@ -0,0 +1,23 @@
+# Branches with names starting with `example` are meant to be long living branches used as
+# documentation/usage example.
+# After accepting the branch, the main PR should be closed
+name: Example branch management
+
+on:
+  pull_request
+
+jobs:
+  example-branches-checks:
+    if: ${{ startsWith(github.head_ref, 'example') }}
+    name: Checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions-ecosystem/action-add-labels@v1
```

Make the action immutable
```suggestion
      - uses: actions-ecosystem/action-add-labels@bd52874380e3909a1ac983768df6976535ece7f8  # v1.1.0
```